.pre-commit-config.yaml

```diff
@@ -12,6 +12,7 @@ exclude: |
   (LICENSE.*|COPYING.*)
 default_language_version:
   python: python3
+  node: "14.13.0"
```

Comment:
?

Reply:
maybe worth to sync the repo with https://github.com/OCA/oca-addons-repo-template ?

Reply:
As we want to avoidbringing all 13.0/wms latest commits on our development branch, I cherry-pick this commit on upstream branch because it is required to get Travis build working (this fix has been done by OCA bot on all OCA repositories).

But the build is still failing anyway... (maybe not related to this commit in particular), perhaps a full rebase is needed.

Reply:
@simahawk PR rebased on OCA/13.0, build fixed